pipeline/data_merging/variant_merging.py

```diff
@@ -705,10 +714,11 @@ def repeat_merging(f_in, f_out):
                     if new_value == old_value:
```

Comment:
I realize this isn't part of the PR, and I'm sure it works fine, but it looks a little funny to me. Why not write it as "if new_value != old_value: (contents of current else block here)"?

Reply:
Yes, indeed, I was also stumbling over this... I tried to be smart and remove it, because it is in principle not necessary. Since we are building a sets of distinct values the `list(set(new_value + old_value))` line later on should already take care of this. However, this changes the order of the fields in general. Per se not a problem, but then it turns out that text fields containing ',' will get scrambled...